packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/deploy.ts

```diff
@@ -20,13 +26,36 @@ type ContractInput = [
 
 const makeUserInput = async (flags, args, env): Promise<UserInput> => {
   if (flags.input) return flags.input as UserInput
+
+  if (flags.rdd) {
+    const rdd = getRDD(flags.rdd)
+    const contractAddr = args[0]
+    const aggregator = rdd[CONTRACT_TYPES.AGGREGATOR][contractAddr]
+    return {
+      maxAnswer: aggregator.maxSubmissionValue,
+      minAnswer: aggregator.minSubmissionValue,
+      decimals: aggregator.decimals,
+      description: aggregator.name,
+      billingAccessController:
+        aggregator.billingAccessController ||
+        flags.billingAccessController ||
+        env.billingAccessController ||
+        '',
+      linkToken: aggregator.linkToken || flags.linkToken || env.linkToken || '',
+      owner: aggregator.owner || env.account,
```

Comment:
Is the owner specified in RDD?

Reply:
adding this to allow for the owner to be defined in RDD, but will fallback to flags or env var if not

Reply:
RDD isn't an arbitrary file with data. It's generated with [RDD tool](https://github.com/smartcontractkit/rddtool) and the `owner` field isn't part of it. Although it falls back to the correct value right now, we cannot know if `owner` field will be added in the future and what would be the meaning of it. It also causes confusion when reading the code. Same applies to the all aggregator fields, e.g. `linkToken` up there.

Reply:
i see, thanks for the context @sdrug  I'll keep it consistent with other parts of the codebase for this PR then, and we can maybe explore updating the RDD tool with additional fields as a later piece of work?

Reply:
Please create a ticket for that and sync with John regarding scheduling it into the sprint. It's a blocker for EngOps if they can't source full data from RDD files.

Reply:
Tracking this work here - https://smartcontract-it.atlassian.net/browse/BCI-1405